src/openforms/contrib/referentielijsten/client.py

```diff
@@ -39,17 +40,23 @@ def get_tabellen(self) -> list[Tabel]:
         all_data = list(pagination_helper(self, data))
         return all_data
 
-    def get_items_for_tabel(self, code: str) -> list[TabelItem]:
-        response = self.get("items", params={"tabel__code": code}, timeout=10)
+    def get_items_for_tabel(self, code: str, current_language: str) -> list[TabelItem]:
+        response = self.get(
+            "items",
+            params={"tabel__code": code},
+            timeout=10,
```

Comment:
can we drop the timeout here? it should the taken from the configuration on the `Service` model and ape-pie already takes care of that